app/views/photos/new.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= form_for @photo, html: {multipart: true} do |f| %>
+  <%= f.label :title %>
+  <%= f.text_field :title %>
+  <%= f.label :description %>
+  <%= f.text_field :description %>
+  <%= f.file_field :image %>
+  <%= f.submit "Add Photo"%>
+<% end %>
```

Comment:
You could extract this `form` to a partial which could then be used to `edit` existing photos too.